resources/tools/INSTALL.txt

```diff
@@ -5,8 +5,8 @@
 PREREQUISITES
 -------------
 
-Ballerina requires JRE 8 update 202.
-Check https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u202-b08 to download the JRE relevant to your system.
+Ballerina requires JRE 11.
+Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.

```

Comment:
```suggestion
Check https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot to download the JRE relevant to your system.
```suggestion
[Download](https://adoptopenjdk.net/releases.html?variant=openjdk11&jvmVariant=hotspot) the JRE relevant to your system.

```
```

Reply:
This is a text file not a MD file